src/views/minor/Footer.vue

```diff
@@ -3,10 +3,21 @@
         <p>
             Thank you very much to
             <a href="/doc/doc/acknowledgments/index.html" class="footer-link">
-                those who made Numberscope possible
-            </a>
+                those who made Numberscope possible</a
+            >
             over the years!
         </p>
+        <p>
+            <i class="bi bi-github"></i>&nbsp;
+            <a class="footer-link" href="https://github.com/numberscope"
+                >GitHub</a
+            >
+            <span id="linkSpacer"></span>
+            <i class="bi bi-twitter"></i>&nbsp;
+            <a class="footer-link" href="https://twitter.com/numberscoper"
+                >Twitter</a
+            >
+        </p>
```

Comment:
If you'd like a more compact footer, it would be OK with me to put the GitHub and Twitter links at the beginning followed by the thanks, just with some horizontal space and no paragraph break, so that the thanks would wrap on narrow screens (like phones).  But it's also fine with me to leave as is.

Reply:
I wouldn't say I have a _strong_ preference one way or the other, but I kind of like having the thanks be prominent and having the social links separate from the thanks.

Reply:
That's fine. Just wanted to float the point because on my screen anyway it feels like vertical space us at a premium.